### DISCUSSION TITLE
Support aria-label to make "weird" Unicode event titles screen-readable

### DISCUSSION THREAD

**Discussion by GreenReaper** [pretalx/pretalx#1819] [Ideas] [answered]
As an event plan designer, I'd like to be able to use ["weird" Unicode text](https://lingojam.com/WeirdTextGenerator) such as Ͳհҽ Ɛղçհąղէҽժ Ƒօɾҽʂէ and 𝓕𝓾𝓻𝓽𝓱𝓮𝓻 𝓢𝓸𝓾𝓽𝓱 for the event title while still retaining screen reader functionality. As such, it'd be nice to be able to specify the aria-label attribute for this element (possibly included in the page's <title> element, too) in order to provide a plain-text version for screen readers.

This request is specifically for the global event title, though I could see it being useful for specific sessions, too.

I appreciate this is a bit of a niche request, but the text does look cooler that way, and it's an effect that can't be fully replicated via fonts unless you find or make your own that looks just right and include it with CSS (presumably hosted externally...).

**Comment by rixx** [comment, marked as answer]
That's out of scope, sorry. We *could* try normalising the even title with `unicodedata.normalize('NFKC', str)`, but that would only cover the event title. What about email subjects, email texts, HTML title elements, and other UI elements like the list of events, social media preview cards, and so on?

If you want a fancy looking event name, you could just as well use a transparent png with the typography you want. The logo replaces the event name at the top of the page, and the event name can still be readable and accessible in all text-based contexts.

**Comment by GreenReaper** [comment]
Fair enough! Our instance is used for editing and displaying a preset schedule, so I wasn't thinking about email, etc.

Thanks for the hint in case I want to try for myself. As you say, the logo picture slot is one way to go, and of course embedding is another possibility for this use-case - we might have done that if the main site's host supported it.
